repspark_at/test_cases/login.py

```diff
@@ -6,11 +6,19 @@
 
 class AppLogin(SetUpClass):
     def test_login(self):
-        LoginPage.set_user_name(self, "qwerty@mail.com")
-        LoginPage.set_password(self, "root")
-        LoginPage.click_login_button(self)
+        login_page = LoginPage(self.driver)
+        login_page.set_user_name("qwerty@gmail.com")
+        login_page.set_password("root")
+        login_page.click_login_button()
+
+        # LoginPage.set_user_name(self, "qwerty@mail.com")
```

Comment:
Please remove all commented code (it's a trash).

Reply:
@extsoft 
done